packages/terra-dropdown-button/tests/jest/DropdownButton.test.jsx

```diff
@@ -87,7 +98,7 @@ describe('Dropdown Button', () => {
   it('correctly applies the theme context className', () => {
     const wrapper = mountWithIntl(
       <ThemeContextProvider theme={{ className: 'orion-fusion-theme' }}>
-        <DropdownButton label="Primary Option">
+        <DropdownButton label="Primary Option" id="dropDown">
```

Comment:
can we have jest test to ensure button is assigned with value passed by consumer as ID instead of the auto generated ID.

Reply:
Done.